subgraph-js/src/__tests__/buildSubgraphSchema.test.ts

```diff
@@ -133,12 +133,12 @@ directive @requires(fields: _FieldSet!) on FIELD_DEFINITION
 
 directive @provides(fields: _FieldSet!) on FIELD_DEFINITION
 
-directive @extends on OBJECT | INTERFACE
```

Comment:
Note to reviewers: the reason the order of this changed is due to some code "consolidation" of the refactoring done in `federationSpec.ts`/`federation.ts`. More precisely, the `completeFed1SubgraphSchema` method in `federation.ts` used to manually list the directive, and it did so in an ever so slightly different order than how those same directives are listed in the fed2 spec. But the code now use the same list in both case (the `legacyFederationDirectives` list in `federationSpec.ts`) and that list use the existing order of the fed2 spec, but thus slightly change the order when we're "completing" a fed1 schema.

Now, we could of course avoid that order change here by recreating 2 separate lists, but I genuinely think the change of order here doesn't matter (and so is worth consolidating the code a bit). In particular, I don't think this would show up in "diffs" for users, because this here is the output of `buildSubgraphSchema`, and that's only used by subgraph to generate a valid schema that can be run by Apollo Server, but it is not the schema that user upload to studio in particular (use upload the "input" of `buildSubgraphSchema`).